src/V3Error.h

```diff
@@ -170,8 +173,8 @@ class V3ErrorCode final {
             "INCABSPATH", "INFINITELOOP", "INITIALDLY", "INSECURE",
             "LATCH", "LITENDIAN", "MODDUP",
             "MULTIDRIVEN", "MULTITOP","NOLATCH", "NULLPORT", "PINCONNECTEMPTY",
-            "PINMISSING", "PINNOCONNECT",  "PINNOTFOUND", "PKGNODECL", "PROCASSWIRE",
-            "RANDC", "REALCVT", "REDEFMACRO",
+            "PINMISSING", "PINNOCONNECT",  "PINNOTFOUND", "PKGNODECL", "PRAGMAERR", "PROCASSWIRE",
+            "PROTECTED", "PROTECTERR", "RANDC", "REALCVT", "REDEFMACRO",
```

Comment:
Add the errors to pretendError() below please.

Reply:
OK.